### ISSUE TITLE
The loss function is inconsistent with the  original paper?

### ISSUE THREAD

**Issue by lisir-boy** [mperezcarrasco/Deep-SVDD#1]
Your code does not seem to reflect the loss function proposed in the original paper.Maybe I didn't find it.Can you tell me where they are? Thanks！

**Comment by mperezcarrasco** [comment]
Hi. In the paper, the authors proposed two different objectives: 
- Soft-boundary Deep SVDD (Eq. 3).
- Deep One-Class SVDD (Eq. 4).

As can be seen in the paper results (Table 1), the objective shown in Eq. 3 never gives better results than using Eq. 4, so I only implemented Eq. 4. This objective is in line 98 of train.py file.
